source/_integrations/vizio.markdown

```diff
@@ -81,21 +71,21 @@ To add your Vizio TV to your installation, add the following to your `configurat
 # Example configuration.yaml entry
 media_player:
   - platform: vizio
-    host: IP_ADDRESS
-    access_token: AUTH_TOKEN
+    host: "DEVICE_IP:DEVICE_PORT"
+    access_token: YOUR_AUTH_TOKEN
 ```
 
 {% configuration %}
 host:
-  description: IP address of your device.
+  description: `IP Address:Port` for your device (port is optional but recommended).
```

Comment:
```suggestion
  description: "`IP Address:Port` for your device (port is optional but recommended)."
```

Reply:
🤦‍♂ thanks @frenck this is why I shouldn't submit PR's right before bedtime